moc_sprint_tools/cardwrapper.py

```diff
@@ -14,7 +14,7 @@ def __getattr__(self, k):
 
     def __repr__(self):
         title = self.title.splitlines()[0][:40].strip()
-        return f'<title="{title!r}" priority={self.priority} id={self.id}>'
+        return f'<title="{title}" priority={self.priority} id={self.id}>'
```

Comment:
We're not using priority in our workflow yet, right?

Reply:
The `sort-cards-by-priority` task uses it, which right now just ensures that some notes consistently float to the top of the notes column.